spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/zuul/filters/ZuulProperties.java

```diff
@@ -57,6 +57,10 @@
 
 	private boolean ignoreLocalService = true;
 
+	private boolean isRegexMapper = false;
+
+	private RegexMapper regexMap = new RegexMapper("(?<name>.*)-(?<version>v.*$)", "${version}/${name}");
```

Comment:
`s/regexMap/regexMapper`. Shortening it confused me, thought it was an actual java `Map` of something.